### PR TITLE
v5.0.0-beta.5 - Import typography design tokens into Fozzie

### PR DESCRIPTION
**Pull request by 1itvinka**
### Changed
- Updated type-map with new typography variables for font-size and line-height.
- Typography variables transitioned:
  - `$font-weight-base` > `$font-weight-regular`
  - `$font-weight-bold` definition was removed as it now comes from pie-design-tokens.

## UI Review Checks

- [ ] This PR has been checked with regard to our brand guidelines

### CODE REVIEW

File: src/scss/settings/_variables.scss [hunk 1]
```diff
@@ -9,57 +9,55 @@
...
+        default: ($font-size-h),
+        narrow: ($font-size-g)
     )
 ) !default;
 
-$font-weight-base           : 400;
-$font-weight-bold           : 600;
-$font-weight-headings       : 600;
+$font-weight-headings       : $font-weight-bold;
 $line-height-base           : line-height();
```
Comment (1itvinka) [inline src/scss/settings/_variables.scss:57]: not sure if I should also change this line

File: src/scss/settings/_variables.scss [hunk 2]
```diff
@@ -9,57 +9,55 @@
...
 
 // Font stacks
-$font-family-base           : 'JustEatBasis', Arial, sans-serif;
+$font-family-base           : $font-family-primary, Arial, sans-serif;
 $font-family-mono           : Menlo, Monaco, 'Courier New', monospace;
```
Comment (1itvinka) [inline src/scss/settings/_variables.scss:61]: haven't found this font family in pie design tokens. https://github.com/justeat/pie-design-tokens/blob/master/pie-design-tokens.jsonc

File: src/scss/settings/_variables.scss (hunk 1 above)
Comment (dandel10n) [reply src/scss/settings/_variables.scss:57]: line-height here uses [mixin](https://github.com/justeat/f-utils/blob/10f1daa6b77bd97a1b6083c2cfd2600f5ee2febc/src/scss/functions/_units.scss#L12) so no need to change.

File: src/scss/settings/_variables.scss (hunk 2 above)
Comment (dandel10n) [reply src/scss/settings/_variables.scss:61]: Let's leave it as it is

File: src/scss/settings/_variables.scss [hunk 3]
```diff
@@ -9,57 +9,55 @@
...
-        default: (48, 56),
-        narrow: (32, 40)
+        default: ($font-size-h),
+        narrow: ($font-size-g)
     )
 ) !default;
 
-$font-weight-base           : 400;
-$font-weight-bold           : 600;
-$font-weight-headings       : 600;
```
Comment (DamianMullins) [inline src/scss/settings/_variables.scss:58]: Is the plan to update these references within Fozzie in another PR? If so, do you need to remove them yet?

Comment (dandel10n) [reply src/scss/settings/_variables.scss:58]: `$font-weight-bold` was removed from here as this variable now comes straight from `pie-design-tokens` so all the references can stay as they were.  And as far as I can see all the references for `$font-weight-base` were changed to use `$font-weight-regular` instead (which comes from `pie-design-tokens` as well).

Comment (1itvinka) [reply src/scss/settings/_variables.scss:58]: $font-weight-bold definition was removed as it now comes from pie-design-tokens. I'll update the PR description.
$font-weight-headings is still here (line 56).

**Review by DamianMullins — APPROVED** [review]

**Review by dandel10n — APPROVED** [review]

**Pull request merged**